XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -2318,6 +2318,37 @@
           ]
         }
       ]
+    },
+    {
+      "UniqueId": "TipDocs",
+      "Title": "Tips and Guides",
+      "Subtitle": "",
+      "ImagePath": "",
+      "Description": "",
+      "Items": [
+        {
+          "UniqueId": "CornerRadius",
+          "Title": "CornerRadius",
+          "Subtitle": "Round Corners",
+          "ImagePath": "ms-appx:///Assets/DefaultIcon.png",
+          "Description": "You may set CornerRadius property starting in WinUI 2.2. It sets round corners to your control",
+          "Content": "You may set CornerRadius property starting in WinUI 2.2. It sets round",
+          "IsNew": true,
+          "Docs": [
+            {
+              "Title": "ParallaxView - API",
+              "Uri": "https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Parallaxview"
+            },
+            {
+              "Title": "Guidelines",
+              "Uri": "https://docs.microsoft.com/windows/uwp/design/motion/parallax"
+            }
+          ],
+          "RelatedControls": [
+
+          ]
```

Comment:
Maybe add Button or TabView here since they are controls that use those by default?